### PR TITLE
Feature/descw 1061 button enhanced test

### PR DESCRIPTION
**Pull request by ShawnTurple**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -27,9 +28,11 @@
     },
     "scripts": {
         "production": [
+            "composer validate",
             "Bcgov\\Script\\Checklists::postProductionChecks"
         ],
         "checklist": [
+            "composer validate",
```
Comment (robrien-bcgov) [inline composer.json:35]: had to look this one up. It seems like it should be part of the checklist script in every composer.json, ideally

File: style.css [hunk 2]
```diff
@@ -5,9 +5,9 @@ Author: govwordpress@gov.bc.ca
 Author URI: https://apps.itsm.gov.bc.ca/jira/browse/DESCW-162
 Description: WordPress Block theme is a theme that leverages the full-site editing functionality that is being built in the Gutenberg plugin.
 Requires at least: 5.9
-Tested up to: 6.1
+Tested up to: 6.2
```
Comment (robrien-bcgov) [inline style.css:8]: i'll keep an eye out for style.css on my next wordpress PR. I bet there's references to wordpress 6.1 in a few other places

**Review by mhaswell-bcgov — APPROVED** [review]
Looks good. The filter handles a `size` attribute that isn't tested in a test case here, just wondering if we should have a case for that.

**Comment by ShawnTurple** [comment]
> Looks good. The filter handles a `size` attribute that isn't tested in a test case here, just wondering if we should have a case for that.
it kind of does, but not implicit, it has a default, which is the first use case.  I have another ticket coming up pretty quick, i will review.

**Review by robrien-bcgov — APPROVED** [review]
looks good, and now i kinda get how php test cases work. it's pretty elegant

**Pull request merged**